internal/sirius/warning_types.go

```diff
@@ -0,0 +1,45 @@
+package sirius
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func (c *Client) WarningTypes(ctx Context) ([]RefDataItem, error) {
+	req, err := c.newRequest(
+		ctx,
+		http.MethodGet,
+		"/api/v1/reference-data?filter=warningType",
```

Comment:
I believe that if you use this
```suggestion
		"/api/v1/reference-data/warningType",
```
then it comes back as `[{"handle":"...","label":"..."}]` rather than `{"warningType":[{"handle":"...","label":"..."}]}` which is a little cleaner